backend/src/main/App.ts

```diff
@@ -28,110 +28,120 @@ import { SavedPromotion } from './entity/SavedPromotion';
 /* eslint-disable  no-console */
 /* eslint-disable  @typescript-eslint/no-unused-vars */
 //todo: ormconfig.json should not have synchronize and drop schema as true for production
-createConnection()
-  .then(async (connection) => {
-    const app = express();
-    app.use(bodyParser.json());
-    const PORT = 8000;
-
-    // await loadSampleData();
+export class App {
```

Comment:
There are not any major changes here other than the fact that all these methods have been refactored into the class `App`. Also, `registerHandlersAndRoutes` now has the responsibility of registering handlers and routes. `init` is only responsible for creating the app and listening to the port